lib/SitesManager.php

```diff
@@ -151,8 +151,8 @@ public function getSitesToDisplay() {
 			}
 
 			$site['url'] = str_replace(
-				['{email}', '{uid}', '{displayname}', '{jwt}'],
-				array_map('rawurlencode', [$email, $uid, $displayName, $jwt]),
+				['{email}', '{uid}', '{displayname}', '{jwt}', '{groups}'],
+				array_map('rawurlencode', [$email, $uid, $displayName, $jwt, implode(",", $groups)]),
```

Comment:
Quoting @provokateurin from my last PR:
> I'm afraid that this is not configurable enough. Maybe someone wants to have the groups separate by dashes or semicolons or something else.

Reply:
@provokateurin I agree, that it'd be nice to have this configurable, but comma separated strings are the standard way to serialize such an array of values into a URL. I think it'd be better to support this non-customizable way rather than not having groups in the url at all.

Reply:
What standard would that be? For example OpenAPI already defines multiple "standard" ways: https://swagger.io/docs/specification/serialization/

Reply:
Yes, there's multiple ways, but for "Path parameters" (even mentioned in the given Swagger Doc), this style of serialization is called the "simple" case/variant.